### PR TITLE
fix(ci): setup ci to push the binary to bintray

### LINKED ISSUE #3: Set up CI
the CI system should build a statically linked binary (for use in all linuxes, including alpine) and upload it to bintray. See the https://github.com/deis/workflow-cli for an example of how this is done.

### PR DESCRIPTION
**Pull request by kmala**
fixes #3 
Also removing the vendor directory to keep consistency with our other projects.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,51 @@
 SHORT_NAME := object-storage-cli
 REPO_PATH := github.com/arschles/${SHORT_NAME}
```
Comment (arschles) [inline Makefile:2]: can you change this to be `github.com/deis/${SHORT_NAME}`

Comment (kmala) [inline Makefile:2]: few places in the repo still uses it https://github.com/deis/object-storage-cli/blob/master/actions/download.go#L8.
Created a new issue #5 to track this.

Comment (arschles) [inline Makefile:2]: :+1: thanks

**Comment by arschles** [comment]
I think this PR is worth merging to see if the publishing works

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+deploy:
+  provider: bintray
+  skip_cleanup: true
+  file: _scripts/ci/bintray-ci.json
+  user: deis-admin
+  key:
+    secure: ZazS3VV7K7RsfpUJ6pl9xbRP5yR8TTJmVyamwVThcmcUpCfNtGGVBJdzmu+S3X1eVa9XSgUayLc1yF1HZ0WkB50mZnQQcEsfhSvEnkn2YFAVPOZIn5GYJTDL4llUNImbMz3VB7/a4XC5n+6dnWY4Q76Smqj5kgsfmr7w74PUf21iPyr8F4vkyND2Pli3BdR1AUH9PF5ICZAZcJ8HLZe522YXUuxENCjb8s2tic5OroAAymkhKbmGmogjfBJXAIUu+O+FX01XSoaVn6cE9EXtsraMNDm8LWUwElnqXlfL8I5tZP1omkoiN9cpC128uwwMOV7WzsEGvnSHxv3v43NmYOhjL8OGNn1xsDmtdXpO36evxD1VF8CB3A+p+c/H7sv7lfvRlbd59ye2HBNBI1Wo8PyOP7KJ+mQsh/2Ow9tEQjvainTQ0ca98LS2O2QyVuUSVuknc95wDsugszPGSwjZvHUO6wjl01O/mSybgIVKmQq9QNlUge5KhWpmrL/TnHBS7spzesDEg76qJt4tlTefL2QGknz9IqCoIsiKE6zVOvwJBFs7UQti2c0HThk1KL18qXa8qKHm57AqOct22I8rNyI4MOirv7sCA/GzygraIuQiZNwGBgV7t3XZglXhPNFaP93+X2eNZuXpBu+vierRcy3ay4Un/d4hOHwJpfODgok=
+  on:
+    branch: master
+notifications:
```
Comment (vdice) [inline .travis.yml]: we don't need this until we have a need for a jenkins job for this repo -- go ahead and create a ticket for that work if it's needed.

**Pull request merged**